Spanner/src/ValueMapper.php

```diff
@@ -330,6 +345,14 @@ private function paramType(
         $definition = null,
         $allowMixedArrayType = false
     ) {
+
+        // if the given type maps to a custom class
```

Comment:
```suggestion
        // if the given type maps to a custom class
```

Reply:
I'm not sure I see the difference, but if it's the capitalisation of the first word, then I've taken care of it :) Thanks.